docs/using-radicle/creating-and-sharing-projects.md

```diff
@@ -17,7 +17,7 @@ At the moment, [projects can only be created through the Upstream client](unders
 
 ## Sharing projects
 
-Projects can be shared with their projectID. The projectID can be found under the project name on the project page. 
+Projects can be shared with their projectID. The projectID can be found under the project name on the project page.
```

Comment:
```suggestion
Projects can be shared with their project URN containing the id. The URN can be copied from the project page.
```

Reply:
Can we just say: "Projects can be shared with their project URN."? I'm not sure I understand why we need to say "containing the id", as the `URN` *is* the id.

Reply:
We don't have to say the containing part, just wanted to give an alternative that doesn't remove the notion of ID completely. Your alternative sounds well fine!